### DISCUSSION TITLE
漢化完成後會有缺字變成 = 的問題

### DISCUSSION THREAD

**Discussion by jeff556603** [GpointChen/FFXIVChnTextPatch-GP#383] [Q&A]
請問有人漢化後出現某些字體變成 = 嗎?
想尋問該如何解決

創角時
![圖片](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/145322326/377b2dcc-eb42-4a0d-81b2-cc7d164987a6)
人物介紹
![圖片](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/145322326/b09016fb-c385-4386-9617-53010eee2d7b)

![圖片](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/145322326/2285e065-4e25-452e-bd49-bd149d3213c2)

**Comment by GpointChen** [comment]
正常，預設的字體包會有這種現象

可以額外下載高畫質包

詳情可參考：https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94

**Reply by jeff556603** [reply]
謝謝，成功解決了
